3/debian-10/rootfs/opt/bitnami/scripts/libzookeeper.sh

```diff
@@ -247,14 +257,26 @@ zookeeper_generate_conf() {
     zookeeper_conf_set "${ZOO_CONF_DIR}/log4j.properties" zookeeper.console.threshold "$ZOO_LOG_LEVEL"
 
     # Add zookeeper servers to configuration
+    idFlag=false
+    if [[ "$ZOO_SERVERS" == *"::"* ]]; then
+        idFlag=true
+    fi
```

Comment:
Same suggestion here:
```suggestion
    server_id_with_jumps=no
    [[ "$ZOO_SERVERS" == *"::"* ]] && server_id_with_jumps=yes
```